README.md

```diff
@@ -124,7 +124,7 @@ try {
     $mail->send();
     echo 'Message has been sent';
 } catch (Exception $e) {
-    echo 'Message could not be sent.';
+    echo 'Message could not be sent. ';
```

Comment:
I agree that's not optimal - but it might be better to add a line break, or simpler, merge these two lines into one.

Reply:
Correct, I do agree on the line break, however it would be even better if we added a bold for the preprogrammed messages and then let the error message come after, like this:

echo '<b>Message could not be sent. </b>';

:)

Reply:
Bold doesn't exist in plain text!

Reply:
I added <br> tags but they didn't show up haha

Reply:
No, don't add tags - it's meant to work in plain text. I suggest changing it to:

    echo 'Message could not be sent. Mailer Error: ', $mail->ErrorInfo;

That will produce exactly the same output as your original PR, but it's slightly more efficient.